docs/android/get-started/installation/openjdk.md

```diff
@@ -9,47 +9,46 @@ ms.author: vyedin
 ms.date: 07/22/2018
 ---
 
-# Microsoft's Mobile OpenJDK Distribution Preview
+# Microsoft's Mobile OpenJDK Distribution
 
-_This guide describes the steps for switching to the preview release of Microsoft's distribution of the OpenJDK. This distribution is intended for mobile development._
-
-![Preview Feature](~/media/shared/preview.png)
+_This guide describes the steps for switching to an internal distribution of OpenJDK. This distribution is intended for mobile development._
 
 ## Overview
 
-Beginning with Visual Studio 15.9 and Visual Studio for Mac 7.7, Visual Studio Tools for Xamarin will move from Oracle’s JDK to a **lightweight version of the OpenJDK that is intended solely for Android development**:
-
-![New workflow offering a web preview of OpenJDK in VS 15.8 Preview 5](openjdk-images/openjdk.png)
+Beginning with Visual Studio 15.9 and Visual Studio for Mac 7.7, Visual Studio Tools for Xamarin has moved from Oracle’s JDK to a **lightweight version of the OpenJDK that is intended solely for Android development**. This is a required migration as Oracle is ending support for commercial distribution of JDK 8 in 2019, and JDK 8 is a required dependency for all Android development.
 
 The benefits of this move are:
 
 - You will always have an OpenJDK version that works for Android development.
 
-- Downloading JDK 9 or 10 won’t affect the development experience.
+- Downloading JDK 9+ won’t affect the development experience.
```

Comment:
Can we clarify here that it's Oracle's JDK? Also, "9 or newer" or "9 or greater" would be better than "9+"